src/main/java/mil/dds/anet/utils/AuthUtils.java

```diff
@@ -93,9 +116,21 @@ public static boolean isSuperuser(Person user) {
         || position.getType() == PositionType.ADMINISTRATOR);
   }
 
+  public static boolean isNotRegularSuperUser(Person user) {
```

Comment:
Method name is a bit odd (and can be confusing: for a normal, unprivileged user, is  `isNotRegularSuperUser` true or false?); perhaps rename this to `isEnhancedSuperuser` and reverse the sense of the call.